bundles/org.openhab.binding.nibeheatpump/contrib/NibeGW/Arduino/README.md

```diff
@@ -0,0 +1,112 @@
+# NibeGW Hardware and Compiling hints
+
+
+## RS-485 Modules
+
+For setttings up a NibeGW you need a RS485 module. Most cheap modules out there are compatible with 5V voltage and therefore compatible with Arduino based hardware. Mostly you will get one of two commonly used designs:
```

Comment:
ProDiNo boards have RS-485 port build in, so would it make sense to say that if you board doesn't have rs-485 port you need a rs-485 module.

Reply:
Good point - added.